dataframe/dataframe.go

```diff
@@ -656,6 +659,13 @@ func WithTypes(coltypes map[string]series.Type) LoadOption {
 	}
 }
 
+// Use Delimiter other then ',', for example '\t'
```

Comment:
The name of this function should be `WithDelimiter` and the comment should be in the form:

`// WithDelimiter sets the csv delimiter other than ',', for example '\t'`